src/os/bluestore/BlueStore.cc

```diff
@@ -7085,14 +7163,21 @@ int BlueStore::_mount()
   return r;
 }
 
+//===================================================
+/// GBH place holder
+//==================================================
+
 int BlueStore::umount()
 {
+  dout(5) << __func__ << "::NCB::entered" << dendl;
```

Comment:
guess this line is only for debugging purpose? do you plan to drop it in a future revision in this PR ?